### ISSUE TITLE
Loaders showing up in tablist after server restart

### ISSUE THREAD

**Issue by nicuzzodiddio** [BG-Software-LLC/WildLoaders#113]
### Minecraft's Version

paper 1.8.9

### Plugin's Version

2024.3

### Describe the bug

When a player places a loader, everything works fine. When i stop the server and restart it, the loader will always be into tablist shown as "Loader-(worldname)-(id)" 
![image](https://github.com/user-attachments/assets/71c64427-3a84-4327-8281-b98cdf232bd8)

### To Reproduce

just place a loader and restart the server

**Comment by OmerBenGera** [comment]
Do you use a tablist plugin?

**Comment by OmerBenGera** [comment]
> > Do you use a tablist plugin?
> 
> Tab plugin by neznamy, latest version. It looks like only players with a certain permission see the thing in the tab

Send me a link to the plugin

**Comment by nicuzzodiddio** [comment]
> > > Do you use a tablist plugin?
> > 
> > 
> > Tab plugin by neznamy, latest version. It looks like only players with a certain permission see the thing in the tab
> 
> Send me a link to the plugin

apparently even players without * permission so players with regular permissions will see the thing in the tab, here's the plugin
https://www.spigotmc.org/resources/tab-1-5-1-21-3.57806/

**Comment by nicuzzodiddio** [comment]
any news?

**Comment by OmerBenGera** [comment]
I can't seem to replicate it.
Does it happen if you have NPCs from Citizens, for example?

**Comment by nicuzzodiddio** [comment]
> I can't seem to replicate it. Does it happen if you have NPCs from Citizens, for example?

nope, i have few npcs with citizens and they do not appear in the tab whatsoever

**Comment by OmerBenGera** [comment]
When a player joins the game, I hide all the NPCs from the player
If the TAB plugin does not respect that, it is a bug and should be reported to them

**Comment by nicuzzodiddio** [comment]
> When a player joins the game, I hide all the NPCs from the player If the TAB plugin does not respect that, it is a bug and should be reported to them

ill try without tab plugin and let you know today

**Comment by nicuzzodiddio** [comment]
without Tab plugin, it still had that issue, i found out that the problem was the spigot, which was apparently a bad fork of paper 1.8.8, which is now fixed with another fork.
